outputs.tf

```diff
@@ -13,7 +14,7 @@ output "kube_config" {
 }
 
 output "aks_cluster_node_username" {
-  value = module.aks.cluster_username
+  value = nonsensitive(module.aks.cluster_username)
```

Comment:
And now why are we actually marking this one as nonsensitive. Seems like the consistency on items is a bit off. If needed converse with the group about these items.

Reply:
Same error on this one `Error: Output refers to sensitive values`. I wasn't sure if wanted to mask the username from the log output so added as nonsensitive to keep it as before.
Do you want me to change to sensitive for consistency?

Reply:
Ok, so if that's the reasoning why was the aks_host item made sensitive? Not consistent.

Reply:
Yes, should have kept them consistent from the beginning. I updated this in the latest change for all to be consistent.